make build fails

`make build` fails with
```
Step 16/17 : COPY doc/ testcase/doc/
COPY failed: stat /var/lib/docker/tmp/docker-builder948542279/doc: no such file or directory
make: *** [build] Error 1
```
because `doc` does not exist.

Work-around: Create a `doc/README.md` file so that there is something to copy.

--- a/testcase/doc/README.md
+++ b/testcase/doc/README.md
@@ -1,0 +1,1 @@
+This directory contains documentation of the test case.

